src/mlpack/bindings/python/print_input_processing.hpp

```diff
@@ -306,16 +296,11 @@ void PrintInputProcessing(
       std::cout << prefix << "  " << d.name << "_tuple = to_matrix("
           << d.name << ", dtype=" << GetNumpyType<typename T::elem_type>()
           << ", copy=CLI.HasParam('copy_all_inputs'))" << std::endl;
-      std::cout << prefix << "  " << "if len(" << d.name << "_tuple[0].shape"
+      std::cout << prefix << "  if len(" << d.name << "_tuple[0].shape"
           << ") < 2:" << std::endl;
-      std::cout << prefix << "  " << prefix << d.name
-          << "_reshape = np.reshape(("<< d.name << "_tuple[0]), (" << d.name
-          << "_tuple[0].shape[0] , 1))" << std::endl;
-      std::cout << prefix << "    " << d.name << "_mat = arma_numpy.numpy_to_"
-          << GetArmaType<T>() << "_" << GetNumpyTypeChar<T>() << "(" << d.name
-          << "_reshape, " << d.name << "_tuple[1])" << std::endl;
-      std::cout << prefix << "  " << "else:" << std::endl;
-      std::cout << prefix << "    " << d.name << "_mat = arma_numpy.numpy_to_"
+      std::cout << prefix << "    " << d.name << "_tuple[0].shape = (" << d.name
+          << "_tuple[0].shape[0], 1)" << std::endl;
```

Comment:
Hi @rcurtin I think so we should also add the same for `to_matrix_with_info()`

Reply:
`matrix_with_info` is also using `reshape` for casting 1-D to 2-D matrix

Reply:
You're right, just haven't had a chance to get to it yet.  I'll address your comments tonight---thanks! :+1: